src/Main.elm

```diff
@@ -1,22 +1,75 @@
-module Main exposing (main)
+module Main exposing (Model)
 
+import Accessibility.Aria as A11y
```

Comment:
Very thoughtful of you to use an a11y package, this is something we are struggling to implement 👌🏻

Reply:
As a color blind person myself, I really believe in accessibility and the need for it on the web